README.md

```diff
@@ -1 +1,194 @@
-# .github
\ No newline at end of file
+# **kitexcall**
+
+English | [中文](README_cn.md)
+
+Kitexcall is a command-line tool for sending JSON general requests using kitex, similar to how curl is used for HTTP.
+
+## Features
+
+- **Supports Thrift/Protobuf:** It supports IDL in Thrift/Protobuf formats.
+- **Supports Multiple Transport Protocols:** It supports transport protocols like Buffered, TTHeader, Framed, and TTHeaderFramed, with plans to support GRPC (Protobuf and Thrift Streaming) in the future.
+- **Supports Common Client Options:** It allows specify common client options, such as client.WithHostPorts, etc.
+- **Supports manual data input from the command line and local files: **Request data can be read from command line arguments or local files.
+- **Supports Metadata Passing:** It supports sending transient keys (WithValue) and persistent keys (WithPersistentValue), and also supports receiving backward metadata (Backward) returned by the server.
+- **Supports Receiving Business Custom Exceptions:** It can receive business custom exception error codes, error messages, and additional information.
+- **Supports Multiple Output Formats:** By default, it outputs a human-friendly readable format, and plans to support parseable formats for better integration with other automation tools.
+
+## Installation
+
+```bash
+go install github.com/kitex-contrib/kitexcall@latest
+```
+
+## Usage
+
+### Basic Usage
+
+When using the kitexcall tool, you need to specify several required arguments, including the path to the IDL file, the method name, and the data to be sent. Example:
+
+- Directly specifying request data:
+
+```bash
+kitexcall -idl-path echo.thrift -m echo -d '{"message": "hello"}' -e 127.0.0.1:9999
+[Status]: Success
+{
+    "message": "hello"
+}
+```
+
+- Or reading request data from a file:
+
+```bash
+kitexcall -idl-path echo.thrift -m echo -d '{"message": "hello"}' -e 127.0.0.1:9999 -f input.json
```

Comment:
`-d` not removed?

Reply:
And pls rearrange the example:

1. Place the IDL at the beginning of this Example
2. Instructions on creating a file `input.json` with the contents before showing them the command

Reply:
All fixed